app/i18n/en-US.json

```diff
@@ -85,6 +85,9 @@
   "startup.auth.mfaCode": "Code",
   "startup.auth.mfaDescription": "Your account has been protected with multi factor authentication. Please enter the code from your google authenticator:",
   "startup.auth.message": "Authenticate your account with balloon:",
+  "startup.selective.continue": "Continue",
+  "startup.selective.edit": "Change selection",
+  "startup.selective.message": "By default shares are ignored during the synchronisation. You can change the selection here or anytime later in the settings.",
```

Comment:
I think this message will be a better fit:

Only data which you own personally gets synchronized. If you would like to consume shares as well you need to activate the synchronization for each share.
You may choose additional folders now or at any later time in the settings.

Note: New shares are not synced by default and need to be activated in the settings.

Reply:
As dicussed i used a shorter text.